src/Microsoft.Identity.Web/WebAppExtensions/WebAppCallsWebApiAuthenticationBuilderExtensions.cs

```diff
@@ -35,8 +35,9 @@ public static class WebAppCallsWebApiAuthenticationBuilderExtensions
         /// <remarks>This method cannot be used with Azure AD B2C, as with B2C an initial scope needs
         /// to be provided.
         /// </remarks>
-        public static AuthenticationBuilder AddMicrosoftWebAppCallsWebApi(
-            this AuthenticationBuilder builder,
+        [Obsolete("Rather use MicrosoftAuthenticationBuilder.CallsWebApi")]
```

Comment:
Should this whole class be similar to `WebApiCallsWebApiAuthenticationBuilderExtensions` and still use `AuthenticationBuilder`?

Reply:
We need to look at what the **migration experience** is (from 0.2.*).
When customers migrate an existing app, what should they be told?